### PR TITLE
Add el2g device management commands

### PR DESCRIPTION
**Pull request by doanac**

### CODE REVIEW

File: subcommands/el2g/devices.go [hunk 1]
```diff
@@ -28,6 +30,26 @@ func init() {
...
+		Run:   doAdd,
+		Example: `# Add a device with an SE050 (product ID: 935389312472)
+# Device ID can be found on a device by running:
+#  $ ssscli se05x uid | grep "Unique ID:" | cut -d: -f2
+#  ssscli se05x uid | grep "Unique ID:" | cut -d: -f2
+#  04005001eee3ba1ee96e60047e57da0f6880
+# This ID is hexadecimal and must be prefixed in the CLI with 0x. For example:
+fioctl el2g devices add 935389312472 0x04005001eee3ba1ee96e60047e57da0f6880
+
+# Add a productiondevice with an SE051 HSM (product ID: 935414457472)
```
Comment (vkhoroz) [inline subcommands/el2g/devices.go:47]: typo: productiondevice  -> production device

File: subcommands/el2g/devices.go [hunk 2]
```diff
@@ -28,6 +30,26 @@ func init() {
 		Args:  cobra.ExactArgs(1),
 		Run:   doShow,
 	})
+
+	add := &cobra.Command{
+		Use:   "add <NC12 product-id> <device-id>",
+		Short: "Grant device access to the EdgeLock2GO",
```
Comment (vkhoroz) [inline subcommands/el2g/devices.go:36]: Thanks; this is _waaay_ more clear than a vague "add device to el2go"

File: subcommands/el2g/devices.go [hunk 3]
```diff
@@ -50,6 +50,14 @@ fioctl el2g devices add --production 935414457472 0x04005001eee3ba1ee96e60047e57
 	}
 	add.Flags().BoolVarP(&production, "production", "", false, "A production device")
 	devicesCmd.AddCommand(add)
+	del := &cobra.Command{
+		Use:   "delete <NC12 product-id> <device-id>",
+		Short: "Remove a device from EdgeLock2GO",
```
Comment (vkhoroz) [inline subcommands/el2g/devices.go:55]: IMHO as you have the `grant access to el2go` in the add command; here `revoke access from el2go` seems complementary.

**Review by vkhoroz — APPROVED** [review]
LGTM

just minor rewording suggestions.

File: subcommands/el2g/devices.go (hunk 2 above)
Comment (doanac) [reply subcommands/el2g/devices.go]: https://github.com/foundriesio/fioctl/pull/179/commits/472fd3aff3747f0ade09a0a476382dc63f084d22

File: subcommands/el2g/devices.go (hunk 3 above)
Comment (doanac) [reply subcommands/el2g/devices.go]: https://github.com/foundriesio/fioctl/pull/179/commits/9193a8ffda13fab7d2564178dd9f0220a9d81f49

**Comment by doanac** [comment]
force push with squashed fixes

**Pull request merged**